packages/server/app/Controllers/Http/TracksSearchesController.test.ts

```diff
@@ -17,4 +27,12 @@ test.group('TracksSearchesController', (group) => {
 
         await request.get('/search/track/Biolay').expect(401);
     });
+
+    test('It should fail to search a track as I ve not confirmed my email', async () => {
+        const request = supertest.agent(BASE_URL);
+        const emailIsNotConfirmed = true;
+        await createUserAndAuthenticate(request, emailIsNotConfirmed);
+
+        await request.get('/search/track/Biolay').expect(403);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Do you want to listen to some good Biolay's songs? Please confirm your email first.